CMake/cdat_modules/cython_external.cmake

```diff
@@ -10,7 +10,7 @@ ExternalProject_Add(Cython
   BUILD_IN_SOURCE 1
   CONFIGURE_COMMAND ""
   BUILD_COMMAND ${PYTHON_EXECUTABLE} setup.py build
-  INSTALL_COMMAND ${PYTHON_EXECUTABLE} setup.py install ${PYTHON_EXTRA_PREFIX}
+  INSTALL_COMMAND ${PYTHON_EXECUTABLE} setup.py install --old-and-unmanageable ${PYTHON_EXTRA_PREFIX}
```

Comment:
@aashish24 found this at: http://stackoverflow.com/questions/18300017/how-to-prevent-setuptools-install-package-as-an-egg. Do you want me to put this in this file?